apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -274,15 +281,16 @@ export default class AnimationPickerBody extends React.Component {
               this.multiSelectEnabled_
             )}
         </ScrollableList>
-        {this.multiSelectEnabled_ && (
-          <div style={styles.footer}>
-            <Button
-              text={msg.done()}
-              onClick={onAnimationSelectionComplete}
-              color={Button.ButtonColor.orange}
-            />
-          </div>
-        )}
+        {this.multiSelectEnabled_ &&
+          (searchQuery !== '' || categoryQuery !== '') && (
```

Comment:
This addition just ensures that the "done" button doesn't appear for the category page (since a student isn't actually "done" there).